src/brokers/broker-pods/components/BrokerPodsBreadcrumb/BrokerPodsBreadcrumb.test.tsx

```diff
@@ -52,4 +89,65 @@ describe('BrokerPodsBreadcrumb', () => {
     fireEvent.click(toggle);
     expect(toggle).toHaveAttribute('aria-expanded', 'true');
   });
+
+  it('should navigate back to PodsList page when click on Apply button in both form and yaml editor', async () => {
+    const comp = render(
+      <MemoryRouter>
+        <BrokerPodsBreadcrumb name={name} namespace={namespace} />
+      </MemoryRouter>,
+    );
+    await waitForI18n(comp);
+
+    fireEvent.click(screen.getByTestId('broker-toggle-kebab'));
+    fireEvent.click(screen.getByText('edit_broker'));
+
+    expect(navigate).toHaveBeenCalledWith(
+      `/k8s/ns/${namespace}/edit-broker/${name}?returnUrl=${encodeURIComponent(
+        window.location.pathname,
+      )}`,
+    );
+  });
+
+  it('should open and close the delete broker modal', async () => {
+    const comp = render(
+      <MemoryRouter>
+        <BrokerPodsBreadcrumb name={name} namespace={namespace} />
+      </MemoryRouter>,
+    );
+    await waitForI18n(comp);
+
+    fireEvent.click(screen.getByTestId('broker-toggle-kebab'));
+    fireEvent.click(screen.getByText('delete_broker'));
+    expect(screen.getByText('delete_modal_instance_title')).toBeInTheDocument();
+
+    fireEvent.click(screen.getByText('cancel'));
+    await waitFor(() =>
+      expect(
+        screen.queryByText('delete_modal_instance_title'),
+      ).not.toBeInTheDocument(),
+    );
+  });
+
+  it('should delete broker and navigate to the brokers list', async () => {
+    (k8sDelete as jest.Mock).mockResolvedValue({});
+    const comp = render(
+      <MemoryRouter>
+        <BrokerPodsBreadcrumb name={name} namespace={namespace} />
+      </MemoryRouter>,
+    );
+    await waitForI18n(comp);
+
+    fireEvent.click(screen.getByTestId('broker-toggle-kebab'));
+    fireEvent.click(screen.getByText('delete_broker'));
+    await waitFor(() => screen.getByText('delete'));
+    fireEvent.click(screen.getByText('delete'));
+
+    await waitFor(() => {
+      expect(k8sDelete).toHaveBeenCalledWith({
+        model: AMQBrokerModel,
+        resource: { metadata: { name, namespace: namespace } },
+      });
+      expect(navigate).toHaveBeenCalledWith(`/k8s/ns/${namespace}/brokers`);
+    });
+  });
```

Comment:
These tests aren't related to the apply and cancel buttons navigation PR.

Reply:
sure , i will move to new PR.